examples/basic/packages/ui/package.json

```diff
@@ -1,21 +1,36 @@
 {
-  "name": "ui",
+  "name": "@repo/ui",
   "version": "0.0.0",
-  "main": "./index.tsx",
-  "types": "./index.tsx",
+  "private": true,
   "license": "MIT",
+  "exports": {
+    "./card": {
+      "types": "./src/card.tsx",
+      "development": "./src/card.tsx",
+      "default": "./dist/card.js"
+    },
+    "./code": {
+      "types": "./src/code.tsx",
+      "development": "./src/code.tsx",
+      "default": "./dist/code.js"
+    }
+  },
   "scripts": {
-    "lint": "eslint .",
+    "build": "tsc",
```

Comment:
Now that UI has it's own build, I think we should promote workspace configs in the default example. IMO it makes things like outputs easier to grock and reason about

Reply:
I'm in.